src/Database/Type/JsonType.php

```diff
@@ -102,4 +107,18 @@ public function marshal($value)
     {
         return $value;
     }
+
+    /**
+     * Set json_encode options.
+     *
+     * @param int $jsonOptions Options. Use JSON_* flags. Set `0` to reset.
+     * @return $this
+     * @see https://www.php.net/manual/en/function.json-encode.php
+     */
+    public function setJsonOptions(int $jsonOptions)
```

Comment:
Probably not a big issue, but `setJsonEncodeOptions()` might be clearer since it's only for json_encode.

Reply:
Or setEncodingOptions()

Reply:
setEncodingFlags() ?

Reply:
We don't have any methods with "Flags" in the name, we just using "Options" so `setEncodingOptions()` sounds good.

Reply:
After all a flag is just a technical detail of an option :) And we try to to hide certain implementation of language details behind a more common speaking term.